Documentation/Contributors/BuildGuide/README.md

```diff
@@ -44,9 +47,26 @@ Cesium ships with a simple HTTP server for testing, run `npm start` after buildi
 
 ```
 npm start
-```
+``` 
+
+Then browse to [http://localhost:8080/](http://localhost:8080/). You should see the following:
+
+* **Apps**
+   * **Cesium Viewer** : our core 3D globe viewer with some basic features included
+   * **Hello World** : basic app to [get you started](http://cesiumjs.org/tutorials/cesium-up-and-running/)
+   * **Sandcastle** : live coding app for viewing and modifying source code from [many examples](https://cesiumjs.org/Cesium/Apps/Sandcastle/index.html?src=Hello%20World.html&label=Showcases)
```

Comment:
You're not modifying the source code with Sandcastle.  I would say 'An app for viewing and creating code examples, complete with a live preview'